src/mono/wasm/runtime/method-binding.ts

```diff
@@ -26,7 +26,12 @@ const _signature_converters = new Map<string, Converter>();
 export function _get_type_name(typePtr: MonoType): string {
     if (!typePtr)
         return "<null>";
-    return cwraps.mono_wasm_get_type_name(typePtr);
+    try {
+        return cwraps.mono_wasm_get_type_name(typePtr);
+    }
+    catch (ex) {
```

Comment:
the ` (ex) ` isn't needed since we're not using it. i would also prefer a string that indicates an error occurred, not just "not-found"

Reply:
I'm also not sure it's a good idea to swallow an exception that happened inside cwraps like this. It seems like it would indicate some sort of horrible runtime failure.